### PR TITLE
Write some docs

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
i am beginning a user guide. Will need more work, but I'm currently happy iterating fast to start giving a structure and online presence

### CODE REVIEW

File: doc/encoding.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Skrub provides encoders that represent well open-ended strings or dirty
+categories, eg to replace :class:`~sklearn.preprocessing.OneHotEncoder`:
+
+* :class:`GapEncoder`: infers latent categories and represent the data on
+  these. Very interpretable, sometimes slow
+
+* :class:`MinHashEncoder`: a very scalable encoding of strings capturing
+  their similarities. Particularly useful on large databases and well
+  suited for learners such as trees (boosted trees or random forests)
```
Comment (Vincent-Maladiere) [inline doc/encoding.rst:40]: Should we mention the lack of interpretability for MinHash?

**Review by Vincent-Maladiere — COMMENTED** [review]
Since this is the user guide, what about mentioning scikit-learn CountVectorizer, HashingVectorizer, and TfIdfTransformer? Understanding the tradeoffs and when to select which would bring value to the user IMO. WDYT?

**Comment by jovan-stojanovic** [comment]
I am writing the user guide part on assembling! Will add it to this PR when I'm done if that's ok ?

**Comment by GaelVaroquaux** [comment]
> I am writing the user guide part on assembling! Will add it to this PR when I'm done if that's ok ?

Either this, or we merge and iterate in a new PR (Vincent's comments are very good, I want to address them, but I'm in a mood of improving fast the website).

File: doc/about.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
+
+About
```
Comment (jovan-stojanovic) [inline doc/about.rst:2]: I wonder if we should separate into two files `About` and `Related projects`?
Not a big thing but if we stuck them together it's ignore by the `Next` button:
image
Good to know for future pages, they should each be done separately.

**Comment by jovan-stojanovic** [comment]
> Either this, or we merge and iterate in a new PR (Vincent's comments are very good, I want to address them, but I'm in a mood of improving fast the website).

Fine for me, let's do small fast steps then. Will open another PR.

**Comment by GaelVaroquaux** [comment]
I merged, so that we can iterate. Will address comments in another PR (feel free to actually address them, as my time to work on this is... irregular)

**Pull request merged**